server/src/test/java/org/cloudfoundry/identity/uaa/zone/GeneralIdentityZoneConfigurationValidatorTests.java

```diff
@@ -288,42 +290,36 @@ public void validate_redirect_uri_protocol_list_elements_conform_to_scheme_patte
         assertEquals(zoneConfiguration, validator.validate(zoneConfiguration, mode));
     }
 
+    @Test
+    public void testSamlSignatureAlgorithmValid() throws Exception{
+        signatureAlgorithmsLesserThanGlobalDefault(SHA1, SHA1, true);
+        signatureAlgorithmsLesserThanGlobalDefault(SHA1, SHA256, true);
+        signatureAlgorithmsLesserThanGlobalDefault(SHA1, SHA512, true);
+        signatureAlgorithmsLesserThanGlobalDefault(SHA256, SHA256, true);
```

Comment:
Why is the expected true if the default is the same as configured alg?

Reply:
Sorry, I don't understand. This particular line tests that `SHA256` is allowed in zone Config if globalDefault is `SHA256`